dask_ml/model_selection/_incremental.py

```diff
@@ -564,6 +602,19 @@ def _fit(self, X, y, **fit_params):
         X, y, scorer = self._validate_parameters(X, y)
         X_train, X_test, y_train, y_test = self._get_train_test_split(X, y)
 
+        if isinstance(self.verbose, bool):
+            # Log all the time if False or True.
+            # If True, logger will be configured to pipe to stdout
+            log_freq = 1.0
+        elif isinstance(self.verbose, float):
+            if not (0 < self.verbose <= 1):
+                msg = "verbose={} must satisfy 0 < verbose <= 1"
+                raise ValueError(msg.format(self.verbose))
+            log_freq = self.verbose
+        else:
+            msg = "verbose must be an instance of float or bool. Got type={}"
+            raise TypeError(msg.format(type(self.verbose)))
```

Comment:
Does this match the behavior of scikit-learn? You can't pass an integer like `verbose=1`?